testgrid/config.yaml

```diff
@@ -2495,6 +2498,9 @@ test_groups:
 - name: pull-cluster-api-provider-aws-bazel-test
   gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-cluster-api-provider-aws-bazel-test
   num_columns_recent: 20
+- name: pull-cluster-api-provider-aws-bazel-e2e
+  gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-cluster-api-provider-aws-bazel-e2e
```

Comment:
Hi @krzyzacy,

I noticed the two different log directory paths for periodic and pre-submits. Do post-submits also use `pr-logs` as part of their log path?